Hello Microverse

javier-salinas opened the pull request:
It was needed to create a new Pull Request due to the requirement to change the Branch names. 
The link of the first Pull Request is:
https://github.com/javier-salinas/Hello_Microverse/pull/1

javier-salinas committed:
add html file

javier-salinas committed:
add html file

javier-salinas committed:
add style file

javier-salinas committed:
Update README.md

javier-salinas committed:
Update README.md

Using updated Readme Template of the project.

javier-salinas committed:
Update README.md

javier-salinas committed:
Merge branch 'main' into Setup_Linters

javier-salinas committed:
Merge branch 'main' into Setup_Linters

victorokoroji commented on README.md:
```diff
@@ -5,6 +5,7 @@
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Key Features](#key-features)
+
 - [👤 Author](#authors)
 - [📝 License](#license)
```
- [ ]  Good job adding the recommended readme as suggested by the previous reviewer 👍🏽 . Please do not delete any section of the microverse template (FAQ section is optional) as I see you did that, but rather, kindly edit all the sections of your readme and add information relating to your project so that your repo will look more professional 👍🏽.

victorokoroji reviewed (changes requested):
Hi @javier-salinas  

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

javier-salinas committed:
Update README.md

Updated with the Microverse README Template, according to the change required by the Reviewer.

iambenkis reviewed (approved):
## Hello @javier-salinas                :wave: ,
I am @iambenkis your reviewer for this PR :man_beard:  

Thank you for making the changes requested by the previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 

![](https://media.giphy.com/media/3kuSo744UIPJjcJUEn/giphy.gif)

## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
